src/views/components-default/TheDefaultFooter.vue

```diff
@@ -348,7 +348,11 @@ export default {
           },
           { label: 'MEW topia', link: 'https://www.mewtopia.com/' },
           { label: 'Press Kit', routerLink: 'PressKit' },
-          { label: 'Security Policy', routerLink: 'SecurityPolicy' }
+          { label: 'Security Policy', routerLink: 'SecurityPolicy' },
```

Comment:
Sorry, could not see where the link went. Its actually supposed to be with the social media links on the bottom:
<img width="327" alt="Screen Shot 2021-06-29 at 10 42 49 AM" src="https://user-images.githubusercontent.com/16910687/123843739-15c11900-d8c7-11eb-936b-a4038a8b7926.png">

Reply:
fixed

![Screenshot from 2021-07-02 19-19-43](https://user-images.githubusercontent.com/9893774/124340500-7de35980-db6a-11eb-8426-9224e74c4270.png)